frontend/packages/knative-plugin/src/components/add/event-sources/CamelSection.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import YAMLEditorField from '@console/shared/src/components/formik-fields/YAMLEditorField';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+
+const CamelSection: React.FC = () => {
+  return (
+    <FormSection fullWidth title="CamelSource">
+      <YAMLEditorField name={'yamlData'} />
```

Comment:
You don't need to pass a string inside curly braces.
```suggestion
      <YAMLEditorField name="yamlData" />
```

Reply:
done